README.md

```diff
@@ -82,17 +82,53 @@ node index config=collector.config.json pairs="COINBASE:BTC-USD,BITSTAMP:btcusdt
 
 ## How to install: Docker
 
+### Setup
+Here are the preliminary steps to follow before building and starting your containers.
+
+Create a config file based on the existing template living in the project's root folder:
+`cp config.json.example config.json`
+
+Make sure to modify the `Dockerfile` accordingly, by replacing:
+`COPY  config.json.example ${WORKDIR}`
+by 
+`COPY  config.json ${WORKDIR}`
+
+## Running Docker
 ```
 ➜ docker-compose build
 ➜ docker-compose up -d
 ```
-This will give you a running server on <http://127.0.0.1:3000> with mounted `./data` volume.
+This will give you a running server on <http://127.0.0.1:3000> with mounted `./data` volume for persistence.
 
 See `./env` file for some basic configuration.
 
-Watch logs using `docker logs -f st-server`.
+## InfluxDB
+
+To start using InfluxDB, set `storage=[ influx ]` in `config.json`.
```

Comment:
If I follow the steps without changing anything to the config, and just following the readme this is the case by default (influx is in the config.json which is copied from config.json.example). 

So when running `docker-compose up - d` (and after I fixed the docker-compose.yml error regarding the services.influx.ports value) it's already trying to connect to inlux (and failing)
something must be wrong here..